.ci/ci_job_flags.sh

```diff
@@ -106,13 +106,12 @@ case "${CI_JOB}" in
 		"CRI_CONTAINERD_K8S")
 			export KUBERNETES="yes"
 			;;
-		"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD")
+		"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD"|"CC_CRI_CONTAINERD_K8S"|"CC_SKOPEO_CRI_CONTAINERD_K8S")
 			# Export any CC specific environment variables
 			export CCV0="yes"
 			export UMOCI=yes
-			if [ "${CI_JOB}" == "CC_SKOPEO_CRI_CONTAINERD" ]; then
-				export SKOPEO=yes
-			fi
+			[[ "${CI_JOB}" =~ SKOPEO ]] && export SKOPEO=yes
+			[[ "${CI_JOB}" =~ K8S ]] && export KUBERNETES=yes
```

Comment:
So after many, many test builds and debugging I found that these lines seemed to be causing the bats tests to not run. I'm not sure why and if there is something strange with the unset variables, but when I replaced them with:
```
if [ "${CI_JOB}" =~ SKOPEO ]; then
	export SKOPEO=yes
fi
if [ "${CI_JOB}" =~ K8S ]; then
	export KUBERNETES=yes
fi
```
the tests ran e.g. http://jenkins.katacontainers.io/job/tests-CCv0-ubuntu-20.04-x86_64-CC_CRI_CONTAINERD-PR/81/console
In that case two of the skopeo skips didn't seem to work, but at least it's slight progress? I'd love to know if anyone has a reason why this might make a difference?

Reply:
The other thing I noticed is that this PR is 45 commits behind the `CCv0` branch, which includes a whole bunch of refactors and re-working, so it is possible that this is causing the issue (when I tested my change I did a re-base on latest)

Reply:
Oops my bad, after looking through the latest output I realised I made a typo with my suggested replacement code as `=~` only works inside double square brackets, so it should have been:
```
if [[ "${CI_JOB}" =~ SKOPEO ]]; then
	export SKOPEO=yes
fi
if [[ "${CI_JOB}" =~ K8S ]]; then
	export KUBERNETES=yes
fi
```